capsules/core/src/process_console.rs

```diff
@@ -1266,6 +1268,62 @@ impl<
 
                                     self.cursor.set(index);
                                 }
+                                EscKey::Backspace if cursor > 0 && cursor <= index => {
+                                    // Move the bytes one position to left
+                                    for i in (cursor - 1)..index {
+                                        command[i] = command[i + 1];
+                                        let _ = self.write_byte(command[i]);
+                                    }
+
+                                    // Now that we copied all bytes to the left, we are left over with
+                                    // a dublicate "ghost" character of the last byte,
+                                    // In case we deleted the first character, this doesn't do anything as
+                                    // the dublicate is not there.
```

Comment:
```suggestion
                                    // a duplicate "ghost" character of the last byte,
                                    // In case we deleted the first character, this doesn't do anything as
                                    // the duplicate is not there.
```